bundles.json

```diff
@@ -17,6 +17,7 @@
     "platform/features/plot",
     "platform/features/scrolling",
     "platform/features/events",
+    "platform/features/rtevents",
```

Comment:
I think we should only have one "events" bundle active; I think we should use regular events here, and rtevents for WARP-RP (I will handle making the change to bundles.json for WARP-RP when I merge)